install.sh

```
@@ -263,6 +264,65 @@ setup_cursor() {
+  else
+    tail -c1 "$codex_config" | grep -q $'\n' || printf '\n' >> "$codex_config"
+    printf '\n[mcp_servers.mnemo]\ncommand = "%s"\nargs = ["mcp", "--tools=agent"]\n' "$mnemo_bin" >> "$codex_config"
+    ok "~/.codex/config.toml: mnemo MCP configured"
+  fi
+
+  if grep -q '^codex_hooks\s*=' "$codex_config" 2>/dev/null; then
+    ok "~/.codex/config.toml: codex_hooks already set"
+  elif grep -q '^\[features\]' "$codex_config" 2>/dev/null; then
+    local tmp
+    tmp=$(mktemp)
+    awk '/^\[features\]/{print; print "codex_hooks = true"; next} 1' "$codex_config" > "$tmp" && mv "$tmp" "$codex_config" || rm -f "$tmp"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)